src/components/search/search-result.js

```diff
@@ -58,7 +58,26 @@ class SearchResult extends React.PureComponent {
               </span>
             </div>
 
-            <div className="mb6">{ReactHtmlParser(excerpt)}</div>
+            <div className="mb6 txt-ms">
+              {ReactHtmlParser(excerpt, {
+                transform: (node, i) => {
+                  if (node.name === 'em') {
+                    return (
+                      <em
+                        key={i}
+                        className="txt-underline txt-bold"
+                        style={{
+                          color: '#4264fb'
+                        }}
+                      >
+                        {' '}
+                        {node.children[0].data}
+                      </em>
```

Comment:
@chriswhong I love the idea of highlighting the relevant terms in the search results! I feel like the current styling makes them look more like links rather than matches - what do you think about using the pattern of highlighting the matching search term instead?

Something like this:
<img width="780" alt="image" src="https://user-images.githubusercontent.com/10137547/209016405-15622cac-edcf-43ab-a89d-812e0106939f.png">


```suggestion
                      <mark
                        key={i}
                        className="txt-bold bg-blue-light color-blue-dark"
                      >
                        {' '}
                        {node.children[0].data}
                      </mark>
```